packages/scaffold-ui/src/utils/w3m-email-otp-widget/index.ts

```diff
@@ -71,8 +71,17 @@ export class W3mEmailOtpWidget extends LitElement {
         gap="l"
       >
         <wui-flex flexDirection="column" alignItems="center">
-          <wui-text variant="paragraph-400" color="fg-100">Enter the code we sent to</wui-text>
-          <wui-text variant="paragraph-500" color="fg-100">${this.email}</wui-text>
+          <wui-text variant="paragraph-400" color="fg-100" align="center"
+            >Enter the code we sent to</wui-text
+          >
```

Comment:
Smol pick, for better readability: 
```suggestion
          <wui-text variant="paragraph-400" color="fg-100" align="center">
            Enter the code we sent to
          </wui-text>
```

The formatter not able to do this since these are template literals, but at least I could sugges few VSCode extension might help you while building UIs with Lit:
- https://marketplace.visualstudio.com/items?itemName=bierner.lit-html
- https://marketplace.visualstudio.com/items?itemName=bashmish.es6-string-css

Reply:
Good catch. Thank you